index.js

```diff
@@ -25,11 +25,19 @@ export const config = {
 const usePreset = () => atom.config.get('linter-stylelint.usePreset');
 const presetConfig = () => atom.config.get('linter-stylelint.presetConfig');
 
+function createRange(editor, data) {
+  // data.line & data.column might be undefined for non-fatal invalid rules,
+  // e.g.: "block-no-empty": "foo"
+  // so we convert undefineds to 0, which will pass -1 to the underlying rangeFromLineNumber
+  // which handles it correctly for us
+  return rangeFromLineNumber(editor, (data.line || 0) - 1, (data.column || 0) - 1);
```

Comment:
If `data.line` or `data.column` aren't set you would be passing -1 instead of 0 in, change the `0` to a `1` so it sends in 0 (and thus gets the whole first line, I've opened https://github.com/AtomLinter/atom-linter/issues/87 for fixing that so if a negative line is passed in it just uses the first line).